### PR TITLE
GG20

### PR DESCRIPTION
**Pull request by kealan**
Partial implemention of

One Round Threshold ECDSA with Identifiable Abort https://eprint.iacr.org/2020/540

### CODE REVIEW

**Review by JohnMcW — APPROVED** [review]
Approved

File: python/amcl/mta_zkp.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+INVALID_ECP = 62   # Invalid ECP
+
+def rp_commit(key, mod, m, c, alpha=None, beta=None, gamma=None, rho=None):
+    """ Generate a Range Proof commitment for the message M
+
+    Args::
+        key : Paillier Key used to ecnrypt M
+        mod : Public bit commitment modulus
+        m
+    """
```
Comment (samuele-andreoli) [inline python/amcl/mta_zkp.py:113]: New line

File: python/benchmark/bench_bc_setup.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    time_func("bc_setup       ", fncall, unit="ms")
+
+    fncall = lambda: bc.priv_to_pub(priv)
+    time_func("bc_priv_to_pub ", fncall, unit="us")
+
+    fncall = lambda: bc.setup_prove(rng, priv, id, ad=ad)
+    time_func("bc_setup_prove ", fncall, unit="ms")
+
+    fncall = lambda: bc.setup_verify(pub, proof, id, ad=ad)
+    time_func("bc_setup_verify", fncall, unit="ms")
```
Comment (samuele-andreoli) [inline python/benchmark/bench_bc_setup.py:65]: New line

File: CMakeLists.txt [hunk 3]
```diff
@@ -129,10 +129,12 @@ if(BUILD_BENCHMARK)
   add_subdirectory(benchmark)
 endif()
 
-if(BUILD_PYTHON)
```
Comment (samuele-andreoli) [inline CMakeLists.txt:132]: It looks like the wrappers should be up to date. I might have forgot to push this change

**Review by samuele-andreoli — CHANGES_REQUESTED** [review]
It looks like there's a pycache that made its way onto the changes, and the python wrappers are not built in the cmake. There's a good chance I just forgot to enable them after updating them. I don't have my laptop on me to check it right now, but it seems likely.
Other than that just spotted a couple typos, but those are not an issue

**Pull request merged**